src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -69,13 +71,24 @@
         if (!lastCommitHash.isEmpty()) {
             fileInfos = getEditedFileInfos(config, lastCommitHash);
         } else {
-            getAllFileInfo(config, fileInfos);
+            fileInfos = getAllFileInfo(config, false);
         }
 
         fileInfos.sort(Comparator.comparing(FileInfo::getPath));
         return fileInfos;
     }
 
+    /**
+     * Extracts a list of relevant binary files given in {@code config}.
+     */
+    public static List<FileInfo> extractBinaryFileInfos(RepoConfiguration config) {
+        logger.info(String.format(MESSAGE_START_EXTRACTING_BINARY_FILE_INFO, config.getLocation(), config.getBranch()));
```

Comment:
No need for the extra log, it can give too much clutter to the terminal output

Reply:
Removed.